docs/scripts/reference.py

```diff
@@ -477,13 +471,72 @@ def gen_reference_commands():
     with out_file.open("w", encoding="utf-8") as f:
         f.write(preface + "\n\n")
 
-        for cmd_name, cmd_path in command_modules:
-            cmd_module = import_module(cmd_path)
+        # Document all commands defined by django-components
```

Comment:
Previously, the reference docs for the commands were generated by searching for all commands in `management/commands/` dir.

This now applies only for the deprecated `startcomponent` and `upgradecomponent` commands.

The rest is now all scoped under the `components` top-level command.

So the logic to discover the commands defined by django-components was updated to:
1. Get the class defining the root `components` command.
2. Add documentation entry on this command.
3. Get all subcommands, and repeat.